plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -1291,16 +1291,17 @@ public function maybe_init_wc_analytics() {
 		 * Hook: woocommerce_analytics_allow_tracking.
 		 *
 		 * @since 9.4.0
-		 * @param boolean $is_allowed Indicates if WooCommerce Analytics should be enabled.
+		 * @param boolean $is_allowed Indicates if WooCommerce Analytics is allowed to track.
 		 */
 		if ( ! apply_filters( 'woocommerce_analytics_allow_tracking', false ) ) {
```

Comment:
We are always returning early here because `woocommerce_analytics_allow_tracking` isn't returning the value of `get_option( 'woocommerce_allow_tracking' )`. I notice that we've got `@since 9.4.0` though so I'm assuming this is to be added later?

For testing this PR I used `add_filter( 'woocommerce_analytics_allow_tracking', '__return_true' );`.

Reply:
> otice that we've got @since 9.4.0 though so I'm assuming this is to be added later?


This is because the repo doesn't pass the lint if we use x.x.x. So I set 9.4.0 for now.

Reply:
> We are always returning early here because woocommerce_analytics_allow_tracking isn't returning the value of get_option( 'woocommerce_allow_tracking' )

Yeah. woocommerce_analytics_allow_tracking is not related at all with  get_option( 'woocommerce_allow_tracking' ). I already got feedback for changing the name of that filter. Thanks